### ISSUE TITLE
Add syntax validation rule for AM009 (note syntax)

### ISSUE THREAD

**Issue by bbringhu** [AdobeDocs/adobe-markdown-authoring#38]
If the writer specifies bad note syntax, we currently don't flag an error. Example:

![bad-note](https://github.com/AdobeDocs/adobe-markdown-authoring/assets/6193209/5f91c4a1-7c47-4092-b032-e5811794ab6c)

The validation rules appear in AM009 (see attached JS file).

[am009.js.zip](https://github.com/AdobeDocs/adobe-markdown-authoring/files/11533466/am009.js.zip)

**Comment by bbringhu** [comment]
@geverit4 - Do you know if any other AM or MD rules are disabled by default? I hope it's just 009

**Comment by geverit4** [comment]
@bbringhu -  here is the current Markdownlint Config:

```JSON
"markdownlint.config": {
    "line-length": false,
    "AM001": false,
    "AM011": false,
    "AM014": false,
    "AM015": false,
    "MD003": {
      "style": "atx"
    },
    "MD007": false,
    "MD009": false,
    "MD012": false,
    "MD014": false,
    "MD024": false,
    "MD025": {
      "front_matter_title": ""
    },
    "MD026": false,
    "MD027": false,
    "MD028": false,
    "MD030": {
      "ul_multi": 1,
      "ol_multi": 1
    },
    "MD033": {
      "allowed_elements": [
        "a",
        "b",
        "br",
        "caption",
        "code",
        "col",
        "colgroup",
        "div",
        "em",
        "I",
        "img",
        "li",
        "ol",
        "p",
        "pre",
        "s",
        "span",
        "strong",
        "sub",
        "sup",
        "table",
        "tbody",
        "td",
        "tfoot",
        "th",
        "thead",
        "tr",
        "u",
        "ul"
      ]
    },
    "MD036": false,
    "MD038": false,
    "MD039": false,
    "MD040": false,
    "MD045": false,
    "MD049": false,
    "MD051": false
  },
```

**Comment by bbringhu** [comment]
Fixed in 2.0.8
